### DISCUSSION TITLE
Missing vaccination lot number in certificate

### DISCUSSION THREAD

**Discussion by remsrc** [Digitaler-Impfnachweis/certification-apis#74] [Q&A]
As a general practitioner I am going to certify the covid-19 vacciations of my patients. I was surprised, that the vaccination lot number (charge) is missing as a necessary field. In my opinion it would raise the security of the certificate, because the lot numbers are time and brand dependent and could be checked against validity. Will this be solved?

**Comment by thinkberg** [comment]
The implementation follows the EU regulation, which explicitly excludes that field.

- https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_json_specification_en.pdf
- https://ec.europa.eu/health/ehealth/covid-19_en
